asyncnsq/tcp/reader.py

```diff
@@ -173,3 +173,8 @@ async def _redistribute(self):
     async def _lookupd(self):
         host, port = random.choice(self._lookupd_http_addresses)
         await self._poll_lookupd(host, port)
+
+
+    def close(self):
```

Comment:
gracefully close connections need more work for reader, if you can do it

Reply:
this is the main purpose of this pr QWQ